pasqal_cloud/errors.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, msg: str, e: Optional[HTTPError] = None) -> None:
         else:
             description = data.get("message", "")
 
-        super().__init__(
+        super().__init__(  # noqa: RET503
```

Comment:
Why not just add the return statement?
```suggestion
        return super().__init__( 
```